doc/changelog/10-coqide/18524-coqide.rst

```diff
@@ -0,0 +1,5 @@
+- **Fixed:**
+  Loading dropped files whose name contains special characters
```

Comment:
```suggestion
  Opening a file with drag and drop now works correctly (fixed regression)
```

This was a regression.  In 8.13, drag and drop works fine except for spaces and Unicode characters, but `-` looks fine.

![image](https://github.com/coq/coq/assets/1253341/89b72bae-ecb4-43a3-a96d-aa8163bb42ed)